appengine/flexible/sparkpost/main.py

```diff
@@ -0,0 +1,39 @@
+from flask import flash, Flask, Markup, redirect, render_template, request
+from sparkpost import SparkPost
+from sparkpost.exceptions import SparkPostAPIException
+
+app = Flask(__name__)
+app.secret_key = 'Sshhh. This is a secret!'
+
+sp = SparkPost()
+
+
+@app.route('/')
+def index():
+    return render_template('index.html')
+
+
+@app.route('/send', methods=['POST'])
+def send():
+    recipient = request.form.get('email')
+    try:
+        sp.transmissions.send(
+            use_sandbox=True,
+            from_email='python-appengine@sparkpostbox.com',
```

Comment:
i think we should (optionally) put this in the app.yml to set in env or config; or put it ui to avoid getting following message
> Exceed Sending Limit (sandbox)